packages/shorebird_cli/lib/src/commands/patch/patch_ios_command.dart

```diff
@@ -246,6 +246,14 @@ ${summary.join('\n')}
       },
     );
 
+    final cleanupProgress = logger.progress('Cleaning up');
+    try {
+      await aotFile.delete();
```

Comment:
What are your thoughts on writing it to a temp directory instead? I feel it would be better than to be modifying the user's workspace especially because partial successes would still lead to having the aotFile around.

Reply:
Done.